FRAuthenticator/FRAuthenticator/Storage/KeychainServiceClient.swift

```diff
@@ -291,7 +295,50 @@ struct KeychainServiceClient: StorageClient {
     }
     
     
+    @discardableResult func setPushDeviceToken(pushDeviceToken: PushDeviceToken) -> Bool {
+        if #available(iOS 11.0, *) {
```

Comment:
not a big deal, but we only support iOS 12 and up, so we don't need the check and the fallback